### PR TITLE
fix: add missing file needed for gh actions bcr workflow

### PR DESCRIPTION
**Pull request by kormide**

### CODE REVIEW

File: .github/workflows/create-bcr-entry.mjs [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+  const [projectPath, bcrPath, ownerSlashRepo, tag] = argv;
+  const version = getVersionFromTag(tag);
+
+  const moduleName = getModuleName(resolve(projectPath, "MODULE.bazel"));
+  const bcrTemplatesPath = resolve(projectPath, ".github/workflows/bcr");
+  const bcrEntryPath = resolve(bcrPath, "modules", moduleName);
+  const bcrVersionEntryPath = resolve(bcrEntryPath, version);
+
+  if (!existsSync(bcrEntryPath)) {
```
Comment (jbedard) [inline .github/workflows/create-bcr-entry.mjs:43]: If the method is async any reason not to use the `fs/promises` API?

**Comment by jbedard** [comment]
Is there anything rules_js specific in this script or is it copied from somewhere?

**Comment by kormide** [comment]
> Is there anything rules_js specific in this script or is it copied from somewhere?

This is just copied from `bazel-lib`. This is a temporary solution until the bcr app is up and running.

File: .github/workflows/create-bcr-entry.mjs (hunk 1 above)
Comment (kormide) [reply .github/workflows/create-bcr-entry.mjs:43]: I'd rather not have copies of this file diverge between repos, and it's temporary, so I'm not too concerned about minor improvements like this.

Comment (jbedard) [reply .github/workflows/create-bcr-entry.mjs:43]: Yeah, if this is just copied then don't change anything. I didn't realize it was copied until looking through a bit more.

**Review by jbedard — APPROVED** [review]

**Pull request merged**